File: code_wars/kyu_7/011-Find_the_vowels.py
###### Solution 1
def vowel_indices(word):
    result = []

    for i, letter in enumerate(word.lower()):
        if letter in {"a", "e", "i", "o", "u", "y"}:
            result.append(i + 1)

    return result


###### Solution 2
def vowel_indices(word):
    return [
        i + 1
        for i, letter in enumerate(word.lower())
        if letter in {"a", "e", "i", "o", "u", "y"}
    ]


###### Video Link:
###### Problem Link: https://www.codewars.com/kata/5680781b6b7c2be860000036/train/python

File: code_wars/kyu_7/011-Find_the_vowels.js
// ###### Solution 1
function vowelIndices(word) {
  const vowels = new Set(["a", "e", "i", "o", "u", "y"]);
  let result = [];
  word = word.toLowerCase();

  for (let i = 0; i < word.length; i++) {
    if (vowels.has(word[i])) result.push(i + 1);
  }

  return result;
}

// ###### Solution 2
function vowelIndices(word) {
  const vowels = new Set(["a", "e", "i", "o", "u", "y"]);
  return word
    .toLowerCase()
    .split("")
    .map((ele, ind) => (vowels.has(ele) ? ind + 1 : 0))
    .filter((ele) => ele);
}
// ###### Video Link:
// ###### Problem Link: https://www.codewars.com/kata/5680781b6b7c2be860000036/train/javascript
